ovos_audio/service.py

```diff
@@ -415,31 +420,52 @@ def _resolve_sound_uri(uri: str):
             raise FileNotFoundError(f"{audio_file} does not exist")
         return audio_file
 
+    @staticmethod
+    def _path_from_hexdata(hex_audio, audio_ext=None):
```

Comment:
this should include a docstring including the expected contents/encoding of `hex_audio`